src/tests/electron/tests/framework-no-codecs.test.ts

```diff
@@ -35,5 +37,26 @@ const releaseTests = process.env.RUN_RELEASE_TESTS === 'true';
         it('has error when loading mp3 <audio> in renderer process', async () => {
             expect(await viewContoller.client.getAttribute('#audio', 'data-err')).toEqual('4');
         });
+
+        it('uses expected build during release', async () => {
+            const buildId = await viewContoller.client.execute(() => {
+                return (window as any).process.versions['microsoft-build'];
+            });
+            expect(buildId).toBe(electronBuildId);
+        });
     },
 );
+
+it('electron versions in package.json and build id are updated together', async () => {
```

Comment:
I thought about running this only in PR builds, but I'm not sure. On one hand it'll simplify electron prototyping locally, but on the other it'll defer issues until PR-time.